### PR TITLE
doc(website): add helm upgrade steps into helm repo

### PR DESCRIPTION
**Pull request by akhilerm**
- add helm upgrade steps into helm repo webpage

Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -36,3 +36,29 @@ helm install <your-relase-name> cstor/openebs-cstor --create-namespace
 ```bash
 helm install <your-relase-name> cstor/openebs-cstor
 ```
+
+#### Upgrade OpenEBS CStor
```
Comment (akhilerm) [inline index.md:40]: @shubham14bajpai Should the data plane update steps also be given here?

Comment (prateekpandey14) [reply index.md:40]: We can add a upgrade repo doc link ?

Comment (shubham14bajpai) [reply index.md:40]: We can link this doc here https://github.com/openebs/upgrade/blob/master/docs/upgrade.md

Comment (akhilerm) [reply index.md:40]: Done.

**Review by shubham14bajpai — APPROVED** [review]
lgtm

**Review by prateekpandey14 — APPROVED** [review]
Lgtm

**Pull request merged**